### ISSUE TITLE
specific URLs to support sharing of specific annotations

### ISSUE THREAD

**Issue by KaitlinCCSI** [NRGI/resourcecontracts.org#1062]
It's possible to get specific URLs for a specific annotation of a specific contract when you use the advanced search functionality. See eg this link: 
http://openlandcontracts.org/contract/ocds-591adf-5399370669/view#/pdf/page/66/annotation/6993

However, if you're looking at the contract itself (eg http://openlandcontracts.org/contract/ocds-591adf-5399370669/view#/pdf) and come across an annotation, there's no way to access that specific URL. I wonder if there would be a non-confusing intuitive way for interested users to access a specific URL, or otherwise share a specific annotation, at this point. Doesn't seem like something you could do with clips, either, even if you took multiple steps to try. Logging this as blue-sky feature for indeterminate future.

**Comment by jedm** [comment]
When I try these steps (at least, I think I'm trying the same steps!), I
click on an annotation in Advanced Search results and it takes me from the
results directly to that Annotation, with the Annotation URL in the browser
window, ie,:

searching here:
http://openlandcontracts.org/search?q=assign

yields a match in Annotation as second result first contract shown (Salala
Rubber 1959)

then click on that result and go to here:
http://openlandcontracts.org/contract/ocds-591adf-6304126551/view#/pdf/page/12/annotation/24474

Yes?

**Comment by KaitlinCCSI** [comment]
Yes--those steps were slightly different than mine, but essentially the same as what I described in first para. Hadn't been able to replicate when working directly within a contract (without having accessed through link in advanced search), but now am able to do so (by clicking on hyperlinked Art. number at end of annotation) -- so either was just being buggy when I tried, or I somehow missed this pathway when I was looking. In any event, all fine now. Thanks.
